src/assets/wise5/authoringTool/add-branch/add-branch.component.ts

```diff
@@ -56,14 +61,12 @@ export class AddBranchComponent {
       text: $localize`Tag`
     }
   ];
-  protected componentIdToSelectable: { [key: string]: boolean } = {};
   protected components: any[];
```

Comment:
Only used in this class?
```suggestion
  private components: any[];
```